File: frontend-web/test_fixtures.py
import os
import json
from supabase import create_client

with open('.env.local', 'r') as f:
    lines = f.readlines()
    env = dict(line.strip().split('=') for line in lines if '=' in line)

supabase = create_client(env['NEXT_PUBLIC_SUPABASE_URL'], env['NEXT_PUBLIC_SUPABASE_ANON_KEY'])
data = supabase.table('fixtures').select('*').limit(1).execute()
print(data.data)

File: frontend-web/test_fixtures.js
const { createClient } = require('@supabase/supabase-js');
const fs = require('fs');

const envFile = fs.readFileSync('.env.local', 'utf8');
const envVars = Object.fromEntries(envFile.split('\n').map(line => line.split('=')));
const supabase = createClient(envVars.NEXT_PUBLIC_SUPABASE_URL, envVars.NEXT_PUBLIC_SUPABASE_ANON_KEY);

async function main() {
  const { data, error } = await supabase.from('fixtures').select('*').limit(1);
  console.log(data);
}
main();
